.github/workflows/ci.yaml

```
@@ -0,0 +1,111 @@
+                include:
+                    -   php: '8.0'
+                        composer_options: "--ignore-platform-req php"
+                        experimental: true
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php }}
+                  coverage: none
+                  extensions: pdo_sqlite, gd
+                  tools: cs2pr
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                  dependency-versions: "highest"
+                  composer-options: "${{ matrix.composer_options }}"
+
+            - name: Setup problem matchers for PHPUnit
+              run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            # - name: Run PHPStan analysis
+            #   run: composer run-script phpstan
+
+            - name: Run code style check
+              run: composer run-script check-cs -- --format=checkstyle | cs2pr
+
+            - name: Run test suite
+              run: composer exec phpunit tests
+
+    postgres-tests:
+        needs: tests
+        services:
+            postgres:
+                image: postgres:10
+                ports:
+                    - 5432
+                env:
+                    POSTGRES_PASSWORD: postgres
+                    POSTGRES_DB: testdb
+                options: >-
+                    --health-cmd pg_isready
+                    --health-interval 10s
+                    --health-timeout 5s
+                    --health-retries 5
+        runs-on: "ubuntu-20.04"
+        timeout-minutes: 10
+        continue-on-error: ${{ matrix.experimental }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                experimental: [ false ]
+                php:
+                    - '7.3'
+                    - '7.4'
+                composer_options: [ "" ]
+                include:
+                    -   php: '8.0'
+                        composer_options: "--ignore-platform-req php"
+                        experimental: true
+
+        steps:
+            -   uses: actions/checkout@v2
+
+            -   name: Setup PHP Action
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: none
+                    extensions: pdo_pgsql, gd
+                    tools: cs2pr
+
+            -   uses: "ramsey/composer-install@v1"
+                with:
+                    dependency-versions: "highest"
+                    composer-options: "${{ matrix.composer_options }}"
+
+            -   name: Setup problem matchers for PHPUnit
+                run: echo "::add-matcher::${{ runner.tool_cache }}/phpunit.json"
+
+            -   name: Run integration test suite vs Postgresql
+                run: composer exec phpunit tests
```

Can we already have https://github.com/ibexa/core/pull/3/commits/02cde5fbea8180f84aeaba6129b1659e1301ae34 and
```suggestion
                run: composer run-script test
```

Side note: this is not a correct way to execute integration tests. Those are split into test suites which are configured via different `phpunit*.xml` files.

---

Fixed in fdfec9f0.

---

The initial idea was to keep those tests separate, because currently Travis does all the other tests.